tests/Schema/ComparatorTest.php

```diff
@@ -42,7 +51,7 @@ public function testCompareSame1(): void
 
         $expected             = new SchemaDiff();
         $expected->fromSchema = $schema1;
-        self::assertEquals($expected, Comparator::compareSchemas($schema1, $schema2));
+        self::assertEquals($expected, $this->comparator->compareSchemas($schema1, $schema2));
```

Comment:
Why this change? The method is still static, isn't it?

Reply:
It is but now the comparator class is extended by platform-specific classes and I want to reuse the same test case to test all implementations. See the classes extending this one.